src/core/include/mp-units/framework/unit.h

```diff
@@ -633,7 +633,7 @@ template<Unit From, Unit To>
 
 template<Unit U1, Unit U2>
 [[nodiscard]] consteval Unit auto common_unit(U1 u1, U2 u2)
-  requires(convertible(u1, u2))
+  requires(convertible(MP_UNITS_IS_CONST_EXPR(u1), MP_UNITS_IS_CONST_EXPR(u2)))
```

Comment:
This is strange. The whole purpose of the trailing requires clause is to use argument identifiers in the expression. Otherwise, it is better to move it up before the function signature and use `U1{}` etc.

Reply:
Thats the error from msvc: 
... unit.h(636): error C7608: atomic constraint should be a constant expression

Should I move it up and use U{}?

Reply:
Yes, please do it.

Reply:
done